src/app/components/MediaLoader/types.ts

```diff
@@ -233,11 +236,49 @@ export type SyndicationAresMediaBlock = {
   };
 };
 
+export type TvMediaBlock = {
+  type: 'tvMedia';
+  model: {
+    id: string;
+    subType: 'episode';
+    format: 'Video';
+    title: string;
+    synopses: {
+      short: string;
+      medium: string;
+    };
+    imageUrl: string;
+    embedding: boolean;
+    advertising: boolean;
+    versions: [
+      {
+        versionId: string;
+        types: string[];
+        duration: number;
+        durationISO8601: string;
+        warnings: Record<string, string>;
+        availableTerritories: {
+          uk: boolean;
+          nonUk: boolean;
+          world: boolean;
+        };
+        availableFrom: number;
+        availabilityStatus: string;
+      },
+    ];
+    availability: string;
+    smpKind: string;
+    episodeTitle: string;
+    type: 'audio' | 'video';
```

Comment:
We can use the MediaType type here
```suggestion
    type: MediaType;
```